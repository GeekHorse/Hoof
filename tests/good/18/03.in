new right one two three done
new down four five six done
new down seven eight nine done
value
delete foo word
value
delete cancel
value
save
