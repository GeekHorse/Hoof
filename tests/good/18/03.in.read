new 
ok 
new 
ok 
new 
ok 
ok   seven eight nine 
huh 
ok 
ok   seven eight 
cancel 
ok   seven eight 
ok 
