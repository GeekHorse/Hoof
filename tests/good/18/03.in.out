new right one two three done
new down four five six done
new down seven eight done
